_guides/understandingmetrics.md

```diff
@@ -1,69 +1,76 @@
-
-### <a name="anomalymetrics" class="jumptarget"></a> Anomalies
-Because we're usually measuring model output against reality - we don't have an error metric for anomaly detection. Anomaly detection is a type of model built using *unsupervised* learning. By definition we don't know what *actual* should be. 
-
-#### <a name="pa" class="jumptarget"></a>Percent Anomalies
+---
+title: Understanding Model Metrics
+description: explaining each metric used for various model comparisons
+category: Nexosis Concepts
+subcategory: Build a model
+tags: [reference, metrics, results]
+quick_link: true
+use_codestyles: true
+order: 9
+---
+
+## Understanding Model Metrics
+It is important to have objective measures to evaluate how accurate the model is whenever a new model is created. It turns out that even the word 'accurate' can be controversial - but what we mean here is some way to tell how close your predictive model's forecasts are to reality.
+
+Each model type may utilize different metrics and different metrics can tell you slightly different things about the same model. The Nexosis API returns several metrics for most models so that you can evaluate your model more fully. Let's look at each metric grouped by the model type.
+
+### <a name="regressionmetrics" class="jumptarget"></a> Forecast & Regression
+
+#### <a name="mae" class="jumptarget"></a>Mean Absolute Error
+Describes in absolute units the difference between the predicted and actual target values, on average for a regression or time series model. This metric does not penalize large prediction errors as heavily as Root Mean Squared Error.  Smaller is always better, but what constitutes a good results depends on the context of the specific problem. If you are predicting house prices a MAE of $500 is a very good result. But if you are predicting book prices that would constitute a bad result.
+
+
+#### <a name="mape" class="jumptarget"></a>Mean Absolute Percent Error
+Describes in percentage how far off the forecasts are from actuals, on average, for a time series model. This has the advantage that it is easily interpretable as a percentage error. However, this metric can be skewed if the target has values close to or equal to zero. In that case any error represents a large percentage error. Smaller is always better and as a rough rule of thumb, anything under 20% can probably be considered a good time series model.
+
+#### <a name="mase" class="jumptarget"></a>Mean Absolute Scaled Error
+This scale-free error metric can be used to compare forecast methods on a single time series and also to compare forecast accuracy between time series. This metric is well suited to intermittent-demand series because it never gives infinite or undefined values except in the irrelevant case where all historical data are equal.  Smaller values are better. If the MASE is less than 1 then the forecast error is better than the average one-step naive forecast, computed in-sample.

```

Comment:
I'm not sure that this is accurate. I believe it's scaled based on the naive forecast so that would change depending on the time series.

Reply:
I also don't have a great feeling for intepreting MASE but I got that description from https://robjhyndman.com/papers/foresight.pdf, page 45, under "Scale-free errors"